core/src/main/java/jenkins/model/ParameterizedJobMixIn.java

```diff
@@ -322,6 +344,72 @@ public final String getBuildNowText() {
          */
         Map<TriggerDescriptor,Trigger<?>> getTriggers();
 
+        /**
+         * @deprecated use {@link #scheduleBuild(Cause)}
+         */
+        @Deprecated
+        @Override
+        default boolean scheduleBuild() {
+            return getParameterizedJobMixIn().scheduleBuild();
+        }
+
+        @Override
+        default boolean scheduleBuild(Cause c) {
+            return getParameterizedJobMixIn().scheduleBuild(c);
+        }
+
+        /**
+         * @deprecated use {@link #scheduleBuild(int, Cause)}
+         */
+        @Deprecated
+        @Override
+        default boolean scheduleBuild(int quietPeriod) {
+            return getParameterizedJobMixIn().scheduleBuild(quietPeriod);
+        }
+
+        @Override
+        default boolean scheduleBuild(int quietPeriod, Cause c) {
+            return getParameterizedJobMixIn().scheduleBuild(quietPeriod, c);
+        }
+
+        // omitting scheduleBuild2(int, Action...) since it is defined in SCMTriggerItem (could include less commonly used overloads if desired)
+
+        // cannot offer makeSearchIndex() since it is defined in Job
```

Comment:
This case would still be tricky however, since the defining method is `protected` and we do not really want to force implementations to override it as `public`.